features/gm.ts

```diff
@@ -197,16 +171,16 @@ export class GroupMessageController implements GroupMessageGetter, GroupMessageL
             return decryptedContent;
         }
 
-        const author = PublicKey.FromHex(event.pubkey);
-        if (author instanceof Error) {
-            return author;
-        }
-
         const json = parseJSON<unknown>(decryptedContent);
         if (json instanceof Error) {
             return json;
         }
 
+        const author = PublicKey.FromHex(event.pubkey);
+        if (author instanceof Error) {
+            return author;
+        }
```

Comment:
according to type `event: Parsed_Event<NostrKind.Group_Message>`, we don't need this logic